src/main.c

```diff
@@ -0,0 +1,105 @@
+#include "global.h"
+
+extern int sub_8001C5C();
```

Comment:
Please use functions.h for externs. Just uncomment the function and give it the appropriate declaration. We want to avoid having all these externs in the .c files.